### ISSUE TITLE
feat: F-Droid release

### ISSUE THREAD

**Issue by Inrumpo** [mozzius/graysky#116]
### Describe the feature you'd like to request

I'd like to be able to download Graysky from F-Droid instead of relying on the Google Play Store.

This will allow Android users to avoid Google services.

### Describe the solution you'd like to see

Submit the open source Graysky Android app to the free and open F-Droid app store.

### Additional information

Submitting to F-Droid Quick Start Guide:

**Comment by mozzius** [comment]
wouldn't be opposed but it seems very involved, especially for a react native app

**Comment by mozzius** [comment]
Btw - you can use Obtainium to fetch the APK from github releases

**Comment by licaon-kter** [comment]
Why does this (as Bluesky) say that it "needs" Zulu JDK? Isn't Java... Java?

**Comment by mozzius** [comment]
That's just what the React Native docs recommend, I don't think it matters too much. I was using openjdk in the past

**Comment by licaon-kter** [comment]
There are 1000s of RN apps in F-Droid, all built with OpenJDK from Debian... :shrug:

**Comment by linsui** [comment]
What's the DATABASE_URL? Is that needed to build the app?

**Comment by mozzius** [comment]
That is not needed to build the app. This is a monorepo, the database url is only for the backend

**Comment by linsui** [comment]
How should we handle the purchase part? The react-native-purchase is non-free so we need to remove it.

**Comment by linsui** [comment]
https://gitlab.com/fdroid/fdroiddata/-/merge_requests/20819 MR draft. How can we get the version code?

**Comment by mozzius** [comment]
What do you need exactly? The version of the app is in `apps/expo/package.json`

**Comment by linsui** [comment]
I need the versionCode for the Android app. See https://developer.android.com/studio/publish/versioning. Without the versionCode F-Droid don't know if a new tag has a new version.

**Comment by mozzius** [comment]
Oh ok, I believe that's managed by EAS - the latest is 223

**Comment by licaon-kter** [comment]
That's versionName. We need versionCode too.

```
 $ aapt2 dump badging application-ab5c5de7-2174-4f5a-a84d-25eb57df7ca2.apk | head
package: name='dev.mozzius.graysky' versionCode='223' versionName='1.7.0' platformBuildVersionName='14' platformBuildVersionCode='34' compileSdkVersion='34' compileSdkVersionCodename='14'
sdkVersion:'23'
```

**Comment by mozzius** [comment]
Yeah, it's currently 223 but it's not committed to the repo - it's generated at build time by EAS
